tests/schema_update_test.go

```diff
@@ -1868,7 +1868,7 @@ CREATE TABLE public.book (
 );
 
 `,
-			wantDatabaseMetadata: `{"name": "latestSchema", "schemas": [{"name": "public", "tables": [{"name": "book", "columns": [{"name": "id", "type": "integer", "nullable": true, "position": 1, "hasDefault": true}, {"name": "name", "type": "text", "nullable": true, "position": 2, "hasDefault": true}], "dataSize": "8192"}]}], "collation": "en_US.UTF-8", "characterSet": "UTF8"}`,
+			wantDatabaseMetadata: `{"name":"latestSchema", "schemas":[{"name":"public", "tables":[{"name":"book", "columns":[{"name":"id", "position":1, "hasDefault":true, "nullable":true, "type":"integer"}, {"name":"name", "position":2, "hasDefault":true, "nullable":true, "type":"text"}], "dataSize":"8192"}]}], "characterSet":"UTF8", "collation":"en_US.UTF-8"}`,
```

Comment:
It appears that this line of change is failing the test.

Reply:
Done. It came from inconsistent pb2json marshaling.